src/command/cmd_funcs.c

```diff
@@ -7125,6 +7136,23 @@ cmd_plugins_update(ProfWin* window, const char* const command, gchar** args)
         return TRUE;
     }
 
+    auto_gchar gchar* scheme = g_uri_parse_scheme(args[1]);
+    if (scheme != NULL && (g_strcmp0(scheme, "http") == 0 || g_strcmp0(scheme, "https") == 0)) {
```

Comment:
This would then also be
```suggestion
    if (_http_based_uri_scheme(scheme)) {
```